### PR TITLE
New home work2

### PR DESCRIPTION
**Pull request by itreyser**

### CODE REVIEW

File: src/lesson1/Main.java [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+import lesson2.Cycles;
+
 import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
         firstExercise(12, 4);
         secondExercise();
         thirdExercise();
+        Conditionals.runningMethods();
```
Comment (DHrytsuk) [inline src/lesson1/Main.java]: It's better to create separate `Main` in `lesson2` package

File: src/lesson2/Addition.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public static void fibonacci() {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Сколько вывести чисел Фиббоначи? Введите в консоль: ");
+        int num = scan.nextInt();
+        int firstNum = 0;
+        int secondNum = 1;
+        System.out.printf("Первые %d чисел/ла Фиббоначи: ", num);
+        for (int i = 0; i < num; i++) {
+            System.out.print(firstNum + " ");
+            int nextNum = firstNum + secondNum;
```
Comment (DHrytsuk) [inline src/lesson2/Addition.java]: This variable is excessive

Comment (itreyser) [reply src/lesson2/Addition.java]: done

File: src/lesson1/Main.java (hunk 1 above)
Comment (itreyser) [reply src/lesson1/Main.java]: done

**Review by DHrytsuk — APPROVED** [review]

**Pull request merged**